ext/libp2p/js/01_peer.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+((window) => {
+  const core = window.Deno.core;
+  const ops = core.ops;
+
+  window.__bootstrap.libp2p ??= {};
```

Comment:
Is `??=` helpful here? Can this script be called multiple times?

Reply:
Good question 👍🏻 

The answer - I don't know 🤷🏻‍♂️

I am copying the style used by Deno, see e.g. here:
 
https://github.com/denoland/deno/blob/f3fb8ee18826e66d3896ea864d2bedaed6c79308/ext/fetch/26_fetch.js#L583-L585
```ts
  window.__bootstrap.fetch ??= {};
  window.__bootstrap.fetch.fetch = fetch;
  window.__bootstrap.fetch.handleWasmStreaming = handleWasmStreaming;
```

BTW this part will be reworked soon. Deno is moving away from this pre-ESM style, I am waiting for a new release of deno_core to adopt the new ESM style too.

This is how the code looks like now:

https://github.com/denoland/deno/blob/b15f9e60a040e2e450e7ca9971a5fc07dbf8b94c/ext/fetch/26_fetch.js#L585
```ts
export { fetch, handleWasmStreaming };
```